mdflynn/overlook
# Feature/fetch data

## What is the change?
- Adding apiRequest object to retrieve data
- Spy testing for apiRequest
- Adds html structure for dashboard to display fetch data
- Adds styling to dashboards

## What does it fix?
No fix this PR.

## Is this a fix or a feature?
This is a feature.

## Where should the reviewer start?
Review the entire files:
`api-request.js`
`api-request-test.js`
`base.scss`
`index.html`
## How should this be tested?
Run `npm start` in your terminal and navigate to http://localhost:8080/ to see current site structure.
